### ISSUE TITLE
Deploy failed: cannot handle socket (RuntimeError)

### ISSUE THREAD

**Issue by josuetex** [tongueroo/jets#122]
Ubuntu 18.04
RVM
Rails: 5.2.2
Ruby: 2.5.3

```
 jets deploy                                                               
=> Rails app detected: Enabling Jets Afterburner to deploy to AWS Lambda.
Traceback (most recent call last):
	28: from /home/josue/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `<main>'
	27: from /home/josue/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `eval'
	26: from /home/josue/.rvm/gems/ruby-2.5.3/bin/jets:23:in `<main>'
	25: from /home/josue/.rvm/gems/ruby-2.5.3/bin/jets:23:in `load'
	24: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/exe/jets:14:in `<top (required)>'
	23: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/cli.rb:5:in `start'
	22: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/cli.rb:20:in `start'
	21: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/cli.rb:48:in `boot_jets'
	20: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/core.rb:18:in `boot'
	19: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/booter.rb:7:in `boot!'
	18: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/booter.rb:31:in `turbo_charge'
	17: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/turbo.rb:15:in `charge'
	16: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/turbo/rail.rb:14:in `setup'
	15: from /home/josue/.rvm/gems/ruby-2.5.3/gems/jets-1.4.5/lib/jets/turbo/rail.rb:93:in `copy_rack_project'
	14: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:392:in `cp_r'
	13: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1461:in `fu_each_src_dest'
	12: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1479:in `fu_each_src_dest0'
	11: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1463:in `block in fu_each_src_dest'
	10: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:393:in `block in cp_r'
	 9: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:415:in `copy_entry'
	 8: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1392:in `wrap_traverse'
	 7: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1392:in `each'
	 6: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1393:in `block in wrap_traverse'
	 5: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1392:in `wrap_traverse'
	 4: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1392:in `each'
	 3: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1393:in `block in wrap_traverse'
	 2: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1390:in `wrap_traverse'
	 1: from /home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:418:in `block in copy_entry'
/home/josue/.rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/fileutils.rb:1278:in `copy': cannot handle socket (RuntimeError)

```

**Comment by tongueroo** [comment]
I see, it's having trouble copying socket files. As a workaround, delete the socket files in `tmp` before running `jets deploy`. Will dig into this once have time 👍

**Comment by tongueroo** [comment]
Released in v1.4.6 [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
